manifests/ocs/cluster.yaml

```diff
@@ -0,0 +1,46 @@
+apiVersion: ceph.rook.io/v1
+kind: CephCluster
+metadata:
+  name: rook-ceph
+  namespace: rook-ceph
+spec:
+  cephVersion:
+    image: ceph/daemon-base:latest-nautilus-devel
```

Comment:
maybe we need a documented TODO that this image need to change?

Reply:
Possibly? I haven't seen much guidance on *what* image we should use. So this could be updated once we have proper down/upstream.